opt/scopeserver/dataserver/modules/gserver/GServer.py

```diff
@@ -204,9 +204,24 @@ def get_features(self, loom: Loom, query: str):
         descriptions = {
             x: ""
             for x in collapsedResults.keys()
-            if x[1] not in ["region_gene_link", "regulon_target", "marker_gene", "cluster_annotation"]
+            if x[1]
+            not in ["region_gene_link", "regulon_target", "marker_gene", "cluster_annotation", "annotation_category"]
         }
 
+        def process_descriptions(
```

Comment:
Here's how I might write it in 2 functions (ignoring obvious details)
```python
    def join_feature_descriptions():
        descriptions[(element, element_type)] = ", ".join(toProcess)
```

```python
    def add_result(element, element_type):
        pass # I don't understand what's going on here
```

Reply:
It would also be nice if they had fewer parameters than the original function. It's ok to prepare parameters before giving the result to your function

Reply:
`prepare_feature_descriptions` might also be a name worth considering